### PR TITLE
KFSPTS-31685 update KSR to prevent edit of qualifiers in KSR by parameter

### PR DESCRIPTION
**Pull request by jhulslander**

### CODE REVIEW

File: src/main/java/edu/cornell/kfs/ksr/service/impl/SecurityRequestDocumentServiceImpl.java [hunk 1]
```diff
@@ -138,6 +146,20 @@ protected void buildSecurityRequestRoles(SecurityRequestDocument document) {
         document.setSecurityRequestRoles(requestRoles);
     }
     
+    protected boolean isKSRAllowedToEditRoleQualifersForRole(String roleId) {
```
Comment (db483) [inline src/main/java/edu/cornell/kfs/ksr/service/impl/SecurityRequestDocumentServiceImpl.java:149]: Now that you changes the parameter name, should the method name also be updated? Thank you

File: src/main/webapp/WEB-INF/tags/ksr/securityRequestRoleQualifications.tag [hunk 2]
```diff
@@ -8,8 +8,10 @@
 <c:set var="securityRequestRole" value="${KualiForm.document.securityRequestRoles[securityRequestRoleIndex]}" />
 
 <c:set var="securityRequestRoleQualificationHdr" value="${securityRequestRole.newRequestRoleQualification}" />
+<c:set var="roleQualifierReadOnly" value="${!securityRequestRole.allowKSRToManageQualifications}"/>
```
Comment (db483) [inline src/main/webapp/WEB-INF/tags/ksr/securityRequestRoleQualifications.tag:11]: should roleQualifierReadOnly be renamed to roleQualificationReadOnly?

**Review by db483 — CHANGES_REQUESTED** [review]
Code changes look good. I just had a couple of things to consider. Thank you

File: src/main/webapp/WEB-INF/tags/ksr/securityRequestRoleQualifications.tag (hunk 2 above)
Comment (jhulslander) [reply src/main/webapp/WEB-INF/tags/ksr/securityRequestRoleQualifications.tag:11]: ok

File: src/main/java/edu/cornell/kfs/ksr/service/impl/SecurityRequestDocumentServiceImpl.java (hunk 1 above)
Comment (jhulslander) [reply src/main/java/edu/cornell/kfs/ksr/service/impl/SecurityRequestDocumentServiceImpl.java:149]: OK

**Review by db483 — APPROVED** [review]
Changes look good.

**Pull request merged**